components/TextControls/src/BaseTextAreas/private/MDCBaseTextAreaTextView.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+@class MDCBaseTextAreaTextView;
+@protocol MDCBaseTextAreaTextViewDelegate <NSObject>
+- (void)textAreaTextView:(MDCBaseTextAreaTextView *)textView willBecomeFirstResponder:(BOOL)didBecome;
+- (void)textAreaTextView:(MDCBaseTextAreaTextView *)textView willResignFirstResponder:(BOOL)didResign;
+@end
+
+@interface MDCBaseTextAreaTextView : UITextView
+@property(nonatomic, weak) id<MDCBaseTextAreaTextViewDelegate> textAreaTextViewDelegate;
```

Comment:
Can we add nullability here and throughout?

Reply:
Done